src/components/Form/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect }from 'react';
+import { useDispatch } from 'react-redux';
+
+import { getResult } from '../../redux/actions'
+
+
+const Form = () => {
+
+    const dispatch = useDispatch();
+    const [nameInput, setNameInput] = useState("")
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        dispatch(getResult(nameInput))
+        setNameInput("")
+    }
+
+    const updateInput = e => {        
+        setNameInput(e.target.value);
+    }
+    
+    return (
+        <form onSubmit={handleSubmit} role= "form">
+            <label htmlFor="input" >Enter Username</label>
+            <input type="text" name="input" value = {nameInput} onChange={updateInput}></input>
+            <button>Submit</button>
+        </form>
+    )
+}
+
+export default Form
```

Comment:
Nice form file, again very concise no unneeded comments or console logs hanging around. Very clean and well put together, good work 👍